gen-resourcesdocs/templates/chapter-single-definition.tmpl

```diff
@@ -22,19 +22,21 @@ weight: {{.Metadata.Weight}}
 {{.Description | replace "<" "\\<" | indent 2 | indent .Indent | indent .Indent}}
 {{- end}}
 {{if .TypeDefinition}}
+{{ ""  | indent .Indent | indent .Indent}}  <a name="{{.Type}}"></a>
 {{.TypeDefinition | indent 2 | indent .Indent | indent .Indent}}
 {{end}}
 {{- end}}{{/* range .Fields */}}
 
 {{range .FieldCategories}}
-### {{.Name}}
+### {{.Name}} {#{{.Name}}}
```

Comment:
Maybe add a comment about why the fragment identifier is set?

Reply:
Do you need the set the fragment identifier if the same as the heading?
Hugo should automatically generate the anchors for the headings.

Reply:
See my comment below,I would like to keep the original case of the heading (which is a Resource Name)